src/test/java/ay2122s1_cs2103t_w16_2/btbb/logic/parser/util/ParserUtilTest.java

```diff
@@ -131,6 +134,31 @@ public void parseEmail_validValueWithWhitespace_returnsTrimmedEmail() throws Exc
         assertEquals(expectedEmail, ParserUtil.parseEmail(emailWithWhitespace));
     }
 
+    // Order parsers:
+
+    @Test
+    public void parseIsDone_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> ParserUtil.parseIsDone(null));
+    }
+
+    @Test
+    public void parseIsDone_invalidValue_throwsParseException() {
+        assertThrows(ParseException.class, () -> ParserUtil.parseIsDone(INVALID_IS_DONE));
+    }
+
+    @Test
+    public void parseIsDone_validValueWithoutWhitespace_returnsIsDone() throws Exception {
+        IsDone expectedIsDone = new IsDone(VALID_IS_DONE);
+        assertEquals(expectedIsDone, ParserUtil.parseIsDone(VALID_IS_DONE));
+    }
+
```

Comment:
After changing the case insensitive check for IsDone. Can you add a test here to check that `YES` or `yEs` still parses with no errors